include/wlr/xwayland.h

```diff
@@ -303,7 +310,10 @@ void wlr_xwayland_surface_ping(struct wlr_xwayland_surface *surface);
  *          false if it should be ignored
  */
 bool wlr_xwayland_or_surface_wants_focus(
-	const struct wlr_xwayland_surface *surface);
+	const struct wlr_xwayland_surface *xsurface);
+
 
+enum wlr_xwayland_icccm_input_state wlr_xwayland_icccm_input_state(
```

Comment:
Can we rename `input_state` to `input_model`? That would match the wording used by the spec and makes it easier to search for its meaning.